bdleon/workout-plus
# added bootstrap to navbar and added logo

#### Changes Made
1. 
1. Modified file `NavBar.js` to include `bootstrap Navbar` and functionality.
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout bd-css-navbar
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
  > when logged in you should see a logo on navbar and if you shrink the screen size you should see a hamburger menu with links.
    
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
